resolwe/flow/managers/listener.py

```diff
@@ -606,6 +606,7 @@ def check_critical_load(self):
             self.check_critical_load()
             _, item = ret
             try:
+                logger.debug(__("Got command from executor: {}", item))
```

Comment:
Does this work?

According to the next line, `item` is a `bytes` object that needs to be decoded.